vcd/resource_vcd_vapp_vm_tools.go

```diff
@@ -0,0 +1,1168 @@
+package vcd
+
+//lint:file-ignore SA1019 ignore deprecated functions
```

Comment:
Added a comment. We are using upstream deprecated `d.GetOkExists` in multiple places so that we can differentiate booleans `false` vs `unset` in HCL schema. Also added a link to original issue. The point is that Hashicorp discourages usage of this function, but no replacement yet exists (except an experimental one).